argilla-server/src/argilla_server/api/handlers/v1/datasets/records_bulk.py

```diff
@@ -56,6 +56,8 @@ async def create_dataset_records_bulk(
         ],
     )
 
+    await authorize(current_user, DatasetPolicy.create_records(dataset))
```

Comment:
Where is the counterpart change removing this line from the `CreateRecordsBulk`?